.internal-ci/test/fs-integration/basic.py

```diff
@@ -24,6 +24,7 @@ async def wait_for_account_to_sync(id):
            != account_status.get("result").get("local_block_height")) or (account_status.get("result").get("balance_per_token").get("0").get("pending") != "0")):
         await asyncio.sleep(sleepy_time)
         account_status = await fs.get_account_status(id)
+    await asyncio.sleep(sleepy_time)
```

Comment:
why not just increase the value of `sleepy_time` used on line 25?

Reply:
For one i'm not sure if the conditions are right in that that `while` loop.. It's definitely hacky but having the sleep at the end of the function makes sure it sleeps for at **least** 15sec.. since on line 144-145 it's being awaited twice in a row.